_data/meltano/extractors/tap-messagebird/meltanolabs.yml

```diff
@@ -0,0 +1,53 @@
+capabilities:
+- catalog
+- state
+- discover
+- about
+- stream-maps
+- schema-flattening
+description: An omnichannel communications platform, built for global scale.
+domain_url: https://www.messagebird.com
+keywords:
+- meltano_sdk
+- api
+label: Messagebird
+logo_url: /assets/logos/extractors/messagebird.png
+maintenance_status: development
+name: tap-messagebird
+namespace: tap_messagebird
+next_steps: ''
+pip_url: git+https://github.com/MeltanoLabs/tap-messagebird.git
+repo: https://github.com/MeltanoLabs/tap-messagebird
+settings:
+- description: The token to authenticate against the API service. Test keys are not
```

Comment:
The token to authenticate against the API service. Test keys are not supported for Conversations see https://support.messagebird.com/hc/en-us/articles/360000670709-What-is-the-difference-between-a-live-key-and-a-test-key-